lib/resty/upload.lua

```diff
@@ -93,21 +111,39 @@ function _M.set_timeout(self, timeout)
     return sock:settimeout(timeout)
 end
 
+local function append_body(self, ...)
```

Comment:
I don't think you need to use varargs here, as you only ever have at most 3 args. Also you don't need to create a table with varargs, you can `select("#", ...)` and `select(1, ...)`.